ras_backstage/resources/reporting_units/get_reporting_unit.py

```diff
@@ -68,7 +68,9 @@ def add_collection_exercise_details(collection_exercises, reporting_unit, case_g
         reporting_unit_ce = party_controller.get_party_by_business_id(reporting_unit['id'], exercise['id'])
         exercise['companyName'] = reporting_unit_ce['name']
         exercise['companyRegion'] = reporting_unit_ce['region']
-        exercise['tradingAs'] = f"{reporting_unit_ce['tradstyle1']} {reporting_unit_ce['tradstyle2']} {reporting_unit_ce['tradstyle3']}"
+        exercise['tradingAs'] = f"{reporting_unit_ce['tradstyle1']}" \
```

Comment:
I think you can do something like f'''<your stuff>'''  and split the stuff in the middle over multiple lines.  If not, no worries because I can't remember the syntax off the top of my head

Reply:
I'll check for a better way of doing this, I'm only altering the line because it failed the linting.